### PR TITLE
CLS17-18 exam statistics test functions

### PR DESCRIPTION
**Pull request by 09041985**
@dmitrytokarev Take a look please.

### CODE REVIEW

File: codecademy/exam_statistics.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+def grades_sum(scores):
+    assert isinstance(scores, list), "Passed the wrong data type"
+    j = 0
+    for i in scores:
+        assert isinstance(i, int) and type(i) != bool and i >= 0, "Passed the wrong data type"
```
Comment (dmitrytokarev) [inline codecademy/exam_statistics.py]: please use `type()` only:
```python
        assert type(i) == int and i >= 0, "Passed the wrong data type"
```

File: tests/test_exam_statistics.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
+import sys
+
+import pytest
+
+from codecademy.exam_statistics import grades_sum
+from codecademy.exam_statistics import grades_average
+from codecademy.exam_statistics import grades_variance
```
Comment (dmitrytokarev) [inline tests/test_exam_statistics.py]: You can chain-import from same package (see my comment on your previous PR for details)

**Comment by 09041985** [comment]
@dmitrytokarev Corrected your comments, I totally forgot about this PR :)

**Comment by dmitrytokarev** [comment]
i changed PR title

**Comment by dmitrytokarev** [comment]
Ha, interesting... merging master onto the working branch after the code review approval does not invalidate that approval.

**Pull request merged**